src/item.cpp

```diff
@@ -6198,7 +6198,7 @@ int item::bash_resist( bool to_self ) const
     // Don't give reinforced items +armor, just more resistance to ripping
     const int dmg = damage_level();
     const int eff_damage = to_self ? std::min( dmg, 0 ) : std::max( dmg, 0 );
-    const int eff_thickness = std::max( 1, get_thickness() - eff_damage );
+    const int eff_thickness = std::max( 0.1f, get_thickness() - eff_damage );
```

Comment:
`eff_thickness` is still of `int` type. Is this correct?

Reply:
it is not, thanks.